src/main/java/net/sf/jabref/importer/fileformat/BibTeXMLHandler.java

```diff
@@ -32,17 +31,17 @@
  */
 class BibTeXMLHandler extends DefaultHandler {
 
+    private static final String BIBTEXML_URI = "http://bibtexml.sf.net/";
+
     private List<BibEntry> bibitems;
 
     private BibEntry b; // the entry being read
 
     // XML parsing stuff
     private String currentChars;
 
+
     public List<BibEntry> getItems() {
-        if (bibitems == null) {
-            return Collections.emptyList();
-        }
```

Comment:
It ensures that a user will never not follow the protocol. The protocol of the usage states that first the entries have to be parsed, then the result retrieved. However, this is not explicitly documented. Should we create another exception "DetectedWrongUsageOfTheAPI"?

Initializing the variable before is a work around, then at EndDocument, this variable has to be reset to ensure that parsing of a second document does not lead to entries added to the result of the first run. - I know, that this is an edge case, but also not following the protocol. If we treat the first wrong usage (getResults before parsing), the we also have to tret the second wrong usage (using the parser twice), don't we?